client/src/component/User.js

```diff
@@ -12,17 +11,22 @@ export const User = () => {
 	};
 
 	return (
-		<div className="user-container">
-			<label htmlFor="username">Your GitHub User Name</label>
-			<input
-				className="user-input"
-				type="text"
-				id="username"
-				onChange={handleInputChange}
-			/>
-			<Link to="/result">
+		<div>
+			<h1>Track your pull requests and Codewars rank </h1>
+
+			<div className="user-container">
+				<label htmlFor="username">Your GitHub User Name</label>
+				<input
+				placeholder="Enter your Github User Name ... "
+					className="user-input"
+					type="text"
+					id="username"
+					onChange={handleInputChange}
+				/>
+				<div>
 				<Search />
-			</Link>
+			</div>
+			</div>
 		</div>
```

Comment:
Keep an eye on your formatting, it is difficult to follow where you are closing tags here. If you dont need the wrappinf div you can use a fragment instead <></>